Directory.Packages.props

```diff
@@ -7,18 +7,22 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageVersion Include="Kentico.Xperience.Admin" Version="28.0.0" />
+    <PackageVersion Include="Kentico.Xperience.Azurestorage" Version="28.0.0" />
+    <PackageVersion Include="Kentico.Xperience.Cloud" Version="28.0.0" />
+    <PackageVersion Include="Kentico.Xperience.Imageprocessing" Version="28.0.0" />
     <PackageVersion Include="Kentico.Xperience.WebApp" Version="28.0.0" />
+    <PackageVersion Include="Kentico.Xperience.Core.Tests" Version="28.0.0" />
     <PackageVersion Include="Lucene.Net" Version="4.8.0-beta00016" />
     <PackageVersion Include="Lucene.Net.Analysis.Common" Version="4.8.0-beta00016" />
     <PackageVersion Include="Lucene.Net.Facet" Version="4.8.0-beta00016" />
-    <PackageVersion Include="SonarAnalyzer.CSharp" Version="9.6.0.74858" />
+    <PackageVersion Include="SonarAnalyzer.CSharp" Version="9.18.0.83559 " />
+    <PackageVersion Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.22.0" />
```

Comment:
Is this required to build the solution and run `DancingGoat` or tests?

Reply:
Reversed